bundles/org.openhab.binding.fritzboxtr064/src/main/feature/feature.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="org.openhab.binding.freebox-${project.version}"
+	xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
+	<repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
+
+	<feature name="openhab-binding-fritzboxtr064" description="Fritzbox TR-064 Binding"
+		version="${project.version}">
+		<feature>openhab-runtime-base</feature>
+		<feature>openhab.tp-jaxb</feature>
```

Comment:
Add-ons must only depend on features that are declared in [openhab-core](https://github.com/openhab/openhab-core/blob/master/features/karaf/openhab-core/src/main/feature/feature.xml), but not "tp" features.
It seems that this binding is the very first to require JAXB implementation (and not just API). Is it really required or would there be an easy way to avoid this dependency?

I see that [(very few) other bindings](https://github.com/openhab/openhab-addons/blob/2.5.x/bundles/org.openhab.binding.helios/src/main/feature/feature.xml) directly depend on bundles, which isn't very nice either. We should probably define core features with suitable sets of bundles for certain types of dependencies.

Reply:
... and there seem to be some pretty weird dependencies being drawn in by this - trying to get it work in my IDE I am not at
```
javax.xml.soap.SOAPException: Unable to create message factory for SOAP: Unable to create SAAJ meta-factoryProvider com.sun.xml.internal.messaging.saaj.soap.SAAJMetaFactoryImpl not found
```

Reply:
What java are you running? I‘m using Zulu-11 and it works for windows and linux. But - and I guess that‘s what is different - I‘m using a standard openhab 2.5 installation and remote debugging from IntelliJ IDEA (where compilation works without your changes). I guess the problem is the bnd resolver.

Reply:
The jaxb and jaxws features have been introduced to make a whole bunch of bundles compile for Java 11. If tp-features are not to be used by bindings, they have been put in the wrong place.

Reply:
>  they have been put in the wrong place.

Looks like it. Yes, the tp features are meant for making bundles compile, but they themselves should not be any runtime dependency.

Reply:
How should we fix that? If we rename the feature in core we'll break all bindings using them for the OH3 port build.

Reply:
Let's keep that open until we are close to merging it (what will anyhow only happen to the master branch as we had discussed).